client-v2/package.json

```diff
@@ -113,6 +116,7 @@
     "react-router": "^4.2.0",
     "react-router-dom": "^4.2.2",
     "react-router-redux": "^4.0.8",
+    "react-transition-group": "^2.6.0",
```

Comment:
Can I have it? Can I have it? Pleease? (I'm surprised we don't have it already -- v. handy and also tiny! V. useful when you need to keep components around for animations as they unmount)

Reply:
I just had a flash back to `react-addons` - yes we can have this back!